src/test/java/org/betonquest/betonquest/api/schedule/CronScheduleBaseTest.java

```diff
@@ -0,0 +1,122 @@
+package org.betonquest.betonquest.api.schedule;
+
+import com.cronutils.builder.CronBuilder;
+import com.cronutils.model.Cron;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static com.cronutils.model.field.expression.FieldExpression.always;
+import static com.cronutils.model.field.expression.FieldExpressionFactory.on;
+import static org.betonquest.betonquest.api.schedule.CronSchedule.DEFAULT_CRON_DEFINITION;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * These tests should ensure that the parsing of cron schedules works properly.
+ */
+@SuppressWarnings({"PMD.TooManyStaticImports", "PMD.JUnit5TestShouldBePackagePrivate"})
+public class CronScheduleBaseTest extends ScheduleBaseTest {
```

Comment:
This test test not everything from the CronSchedule class. I think we should test 100% of the API